RecoParticleFlow/PFClusterProducer/python/particleFlowRecHitHBHE_cfi.py

```diff
@@ -31,3 +42,64 @@
 
 )
 
+cuts2017 = particleFlowRecHitHBHE.producers[0].qualityTests[0].cuts
+
+cuts2018 = cms.VPSet(
+    cms.PSet(
+        depth=cms.vint32(1, 2, 3, 4),
+        threshold = cms.vdouble(0.8, 0.8, 0.8, 0.8),
+        detectorEnum = cms.int32(1)
+        ),
+    cms.PSet(
+        depth=cms.vint32(1, 2, 3, 4, 5, 6, 7),
+        threshold = cms.vdouble(0.1, 0.2, 0.2, 0.2, 0.2, 0.2, 0.2),
+        detectorEnum = cms.int32(2)
+        )
+    )
+
+cuts2019 = cms.VPSet(
+    cms.PSet(
+        depth=cms.vint32(1, 2, 3, 4),
+        threshold = cms.vdouble(0.1, 0.2, 0.3, 0.3),
```

Comment:
please point me to the slides that justify this selection.

Reply:
Last week we discussed the criteria for the HE in various forums.
The one on the HB are just a redefinition based on the same number of layer
http://home.fnal.gov/~chlebana/CMS/Phase1/figs/HCAL-Depth-Segmentation-Phase1-HE-HBOption1.pdf

Reply:
Thanks.
It would be nice to eventually have a summary in one place.